pootle/apps/accounts/migrations/0003_remove_pootleprofile_id.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def fix_accounts_alt_src_langs(apps, schema_editor):
+    cursor = schema_editor.connection.cursor()
+
+    # check its mysql - should probs check its not too old
+    if not hasattr(cursor.db, "mysql_version"):
+        return
+
+    # get the db_name and table_name
+    db_name = cursor.db.get_connection_params()['db']
+    table_name = (apps.get_model("accounts.User")
+                  ._meta.local_many_to_many[0].m2m_db_table())
```

Comment:
> I dont think there are any guarantees that the db has the table of a particular name.

Then add a check to ensure the table is in the DB. I think it is much safer to do that instead of what the you are doing now.